skellytracker/process_folder_of_videos.py

```diff
@@ -259,6 +261,8 @@ def get_tracker_params(tracker_name: str) -> BaseModel:
         tracking_params=OpenPoseTrackingParams(
            openpose_root_folder_path=str(openpose_root_folder_path),
            output_json_path=str(output_json_path),
+           track_hands=False,
```

Comment:
Do we want these to be False by default? They're still True as default in the class init

Reply:
Sorry this was a mistake - I was testing how the parsing would break if I set these to false!